src/components/DateTimezoneComponent.tsx

```diff
@@ -0,0 +1,64 @@
+import autobind from "autobind-decorator"
```

Comment:
Added new Component for Date Picker with Timezone Selection as well as Time Selection. react-datepicker component old one, which is not supported with changing timezone. So created a component with the help of new package react-date